README.md

```diff
@@ -1095,6 +1114,25 @@ Then install the new AWX Operator by following the instructions in [Basic Instal
 
 Once the new AWX Operator is up and running, your AWX deployment will also be upgraded.
 
+##### Upgrade of instances with auto upgrade
```

Comment:
I would move this up under the `#### Auto upgrade` section.

Reply:
Moved like suggested